samples/pmic/npmx/buck/README.rst

```diff
@@ -0,0 +1,101 @@
+.. _nPM1300 BUCK:
+
+nPM1300 BUCK
+############
+
+Overview
+********
+
+This sample demonstrates the usage of nPM1300 PMIC BUCKs.
+For testing connect voltage multimeter to VOUT1 (output voltage pin of BUCK1). In this example
+voltage can be changed only for BUCK1. BUCK2 voltage is set by default to 3.0V.
+In main.c file select your test case to be executed.
+You can also modify expected output voltages and selected GPIOs to control the system.
+
+Testcases
+---------
+
+Test set output BUCK voltage
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
```

Comment:
Each testcase starts with "In testcase selection..." - what should I do after wiring to get to this point?